ui/app/models/task-group.js

```diff
@@ -38,6 +38,22 @@ export default class TaskGroup extends Fragment {
     }
   }
 
+  // TODO: This async fetcher seems like a better fit for most of our use-cases than the above getter (which cannot do async/await)
+  async getPathLinkedVariable() {
+    await this.job.variables;
+    if (this.job.parent.get('id')) {
+      return await this.job.variables?.findBy(
+        'path',
+        `nomad/jobs/${JSON.parse(this.job.parent.get('id'))[0]}/${this.name}`
```

Comment:
For my education: why is there a `JSON.parse` here instead of `${this.job.plainId}` (we've done the same in `job.js`, but not in `task.js`)

Reply:
You know what, I don't think there's any good reason. Going to move this over to use plainId.